app/docs/dev/policies/traffic-metrics.md

```diff
@@ -9,7 +9,7 @@ You can add metrics to a mesh configuration, or to an individual data plane prox
 Kuma provides full integration with Prometheus:
 
 * Each proxy can expose its metrics in [Prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/#text-based-format).
-* Because metrics are part of the mesh configuration, We can leverage the control plane api to automatically find every proxy in the mesh.
+* Because metrics are part of the mesh configuration, Kuma exposes an api called the monitoring assigment service (MADS) which exposes every proxy in the mesh.
```

Comment:
```suggestion
* Because metrics are part of the mesh configuration, Kuma exposes an API called the monitoring assignment service (MADS) which exposes every proxy in the mesh.
```

should it be "exposes the metrics of every proxy"? "exposes every proxy" sounds sort of scary

Reply:
The API exposes the proxies and the proxies then in turn expose the metrics